WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -296,6 +296,8 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
         updateGhostableTableViewOptions()
         ghostableTableView.reloadSections([0], with: .automatic)
 
+        postsViewButtonItem.accessibilityLabel = NSLocalizedString("List style", comment: "List display style")
+        postsViewButtonItem.accessibilityValue = isCompact ? NSLocalizedString("Expanded", comment: "Expanded style") : NSLocalizedString("Compact", comment: "Compact style")
```

Comment:
When the button is selected and the list is in compact view, the `accessibilityValue` is “Expanded”. I think we should describe **what the current list style is**. 

<img src="https://user-images.githubusercontent.com/198826/70557697-8d422280-1b40-11ea-8702-3af0ee2407a9.png" width="320">

I think it makes more sense since after you tap on the button, VoiceOver would speak it again with the current `accessibilityValue`. 

```suggestion
postsViewButtonItem.accessibilityValue = isCompact ? NSLocalizedString("Compact", comment: "...") : NSLocalizedString("Expanded", comment: "...")
```

What do you think? 🙂

Reply:
Makes sense! Will do :)

Reply:
This has been taken care of!